src/intents.js

```diff
@@ -46,6 +46,14 @@ function injectService (url, element, intent, data) {
         return event.source.postMessage(data, event.origin)
       }
 
+      if (handshaken && event.data.type === `intent-${intent._id}:size`) {
+        ['width', 'height', 'maxWidth', 'maxHeight', 'minWidth', 'minHeight'].forEach(prop => {
```

Comment:
Oh, I didn't see it like that, so we just need the max properties? If we don't need the min properties we can skip them for now. It's just better to not handle not used yet properties to avoid useless maintenance 👍